CompatBot/Program.cs

```diff
@@ -193,12 +194,17 @@ internal static async Task Main(string[] args)
                                                  }
                                                  Config.Log.Info($"All moderation backlogs checked in {gaArgs.Guild.Name}.");
                                              };
+                    client.GuildDownloadCompleted += async gdcArgs =>
+                    {
+                        UsernameValidationMonitor.SetupGuilds(gdcArgs.Guilds.Select(x=>x.Value).ToList());
+                        await Task.CompletedTask;
```

Comment:
this is completely useless

`return Task.CompletedTask` is needed to satisfy the async interface signature in non-async implementation, that will reduce overhead of the compiled method

awaiting completed task does literally nothing